org/mozilla/jss/nss/SSL.java

```diff
@@ -188,6 +191,24 @@ public static native int ConfigServerSessionIDCache(int maxCacheEntries,
      */
     public static native int AttachClientCertCallback(SSLFDProxy fd) throws Exception;
 
+    /**
+     * Enable recording of alerts in the SSLFDProxy object.
+     *
+     * See also: SSL_AlertReceivedCallback in /usr/include/nss3/ssl.h,
+     *           SSL_AlertSentCallback in /usr/include/nss3/ssl.h
+     */
+    public static int EnableAlertLogging(SSLFDProxy fd) {
+        fd.inboundAlerts = new ArrayList<SSLAlertEvent>();
```

Comment:
@edewata Is there a better option here?

I want a data structure that let's me:

 - Track my position in the list
 - Push new elements
 - Know when a new one has been seen
 - But also lets me access previous elements if I need to.

Up until the last requirement, that's kinda looking like a stock queue (and calling `.pop()` and `.push(...)`) but...